apps/coscrad-frontend/src/app/build-routes.tsx

```diff
@@ -132,6 +115,31 @@ export const buildRoutes = (contentConfig: ConfigurableContent): CoscradRoute[]
                 element: <ListenLivePage />,
             }),
         ],
+        [
```

Comment:
Was there a reason for reordering these? They don't seem to affect photographs in any way, but this change could affect work that Justin is doing.

Reply:
This doesn't have to happen in this PR, but once he's done, it would be good to organize the nav.  Originally we talked about having a predictable order to the nav menu that people expect from visiting other sites.  Mainly it generally goes Home, About, then content stuff and then links and credits come last like in a movie.  I've often seen the listen live link towards the end of a nav menu so that it's easily accessible.  So I could see: 

Home, About, Research Resources, [...], Listen Live, Additional Materials, Links, Credits

Reply:
I'm not opposed to this at all. I just want to keep this PR focused because there's enough going on already. Also,  it's nice to keep the commits for such changes atomic. But let's add a story for this and do it soon!